docs/creating-a-runtime-check.md

```diff
@@ -0,0 +1,209 @@
+# Creating a Runtime Check
+
+Runtime checks are used to perform tests by executing the plugin's code.
+
+The Plugin Checker uses a number of interfaces to build out runtime checks.
+
+The Plugin Checker provides the `Runtime_Check` interface, which is used to identify a runtime check. This interface does not contain any methods but serves as a marker for runtime checks.
+
+```php
+use WordPress\Plugin_Check\Checker\Check_Result;
+use WordPress\Plugin_Check\Checker\Runtime_Check;
+use WordPress\Plugin_Check\Traits\Stable_Check;
+
+class Custom_Check extends Runtime_Check {
+  
+  use Stable_Check;
+  
+  public function get_categories() {
+    // Return an array of check categories.
```

Comment:
See my feedback on the static check code example, here it would be helpful to point to where the available categories are defined.